src/vs/workbench/contrib/testing/browser/testExplorerActions.ts

```diff
@@ -983,11 +983,15 @@ abstract class ExecuteTestAtCursor extends Action2 {
 	public async run(accessor: ServicesAccessor) {
 		const editorService = accessor.get(IEditorService);
 		const activeEditorPane = editorService.activeEditorPane;
-		const activeControl = editorService.activeTextEditorControl;
+		let activeControl = editorService.activeTextEditorControl;
```

Comment:
This PR assumes that the modified editor in the diff editor is always the one where tests are present, but this may not be the case. I think we should probably use `editorService.activeEditor`, since that will be the editor that focus is currently in, and then there's no need to check `isDiffEditor`